packages/next/src/server-build.ts

```diff
@@ -428,7 +451,11 @@ export async function serverBuild({
       }
 
       if (lambdaAppPaths[page]) {
-        appRouterPages.push(page);
+        if (lambdaAppPaths[page].fsPath.endsWith('route.js')) {
+          appRouteHandlers.push(page);
+        } else {
+          appRouterPages.push(page);
+        }
```

Comment:
This change is not behind a flag. Is that intentional?

Reply:
yes, we should have app route handlers into their own lambdas!